fl4health/losses/mkmmd_loss.py

```diff
@@ -0,0 +1,288 @@
+# multi-kernel maximum mean discrepancy
+# cao bin, HKUST, China, binjacobcao@gmail.com
+# free to charge for academic communication
+
+from logging import INFO
+from typing import Optional
+
+import torch
+from flwr.common.logger import log
+from qpth.qp import QPFunction, QPSolvers
+
+
+class MkMmdLoss(torch.nn.Module):
+    def __init__(
+        self,
+        device: torch.device,
+        gammas: Optional[torch.Tensor] = None,
+        betas: Optional[torch.Tensor] = None,
+        minimize_type_two_error: bool = True,
+    ) -> None:
+        """
+        Compute the multi-kernel maximum mean discrepancy (MK-MMD) between the source and target domains. Also allows
+        for optimization of the coefficients, beta
+
+        Args:
+            device (torch.device): Device onto which tensors should be moved
+            gammas (Optional[torch.Tensor], optional): These are known as the length-scales of the RBF functions used
+                to compute the Mk-MMD distances. The length of this list defines the number of kernels used in the
+                norm measurement. If none, a default of 29 kernels is used. Defaults to None.
```

Comment:
Is the 29 kernels still correct after shifting the default?

Reply:
It is actually 19.